test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -365,24 +365,24 @@ def test_mom_dyn_res_permissions(self):
 
         # give write permission to group and others
         self.du.chmod(path=fp, mode=0766)
-        self.check_log(fp)
+        self.check_access_log(fp)
 
         # give write permission to group
         self.du.chmod(path=fp, mode=0764)
-        self.check_log(fp)
+        self.check_access_log(fp)
 
         # give write permission to others
         self.du.chmod(path=fp, mode=0746)
-        self.check_log(fp)
+        self.check_access_log(fp)
 
         # give write permission to user only
         self.du.chmod(path=fp, mode=0744)
-        self.check_log(fp, exist=False)
+        self.check_access_log(fp, exist=False)
 
-        # Cleanup
-        os.remove(fp)
```

Comment:
You want to add this to the file list as close to the create_temp_file() as possible.